docs/UserGuide.md

```diff
@@ -293,10 +293,15 @@ Locates and ranks the nearest gyms to user's location in NUS.
 
 Format: `locate LOCATION`
 
+Parameters: COM2 | S13 | S17 | CLB | UHC | LT1 | LT9 | AS6
```

Comment:
Is this supposed to be rendered as a table?

<img width="771" alt="Screenshot 2022-11-01 at 3 54 06 PM" src="https://user-images.githubusercontent.com/97275050/199185513-4109032a-8f3f-4e91-a31b-bc5945b8c2d3.png">